src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj

```diff
@@ -16,7 +16,7 @@
 		<RootNamespace>Umbraco.Community.BlockPreview</RootNamespace>
 		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
 
-		<VersionPrefix>1.8.2</VersionPrefix>
+		<VersionPrefix>2.0.0</VersionPrefix>
```

Comment:
Leave this set as 1.8.2 for the time being

Reply:
Ah It was from my first version! where I was navigating through service to figure out if it si block. let me correct it :)